packages/aila/src/features/moderation/AilaModeration.ts

```diff
@@ -144,6 +146,7 @@ export class AilaModeration implements AilaModerationFeature {
   public reportModerationToAnalytics(
     moderationResult: ModerationResult,
     moderation: Moderation,
+    durationMs?: number,
```

Comment:
Is this method used elsewhere? I'm thinking why not make durationMs non-optional?

Reply:
Yeah, my bad. I'm going to close this for now anyway and look at telemetry for this